build/parsePrep.c

```diff
@@ -273,6 +273,15 @@ static int doSetupMacro(rpmSpec spec, const char *line)
 	free(buf);
     }
 
+    /* mkdir for dynamic specparts */
+    buf = rpmExpand("%{__mkdir} SPECPARTS", NULL);
+    appendBuf(spec, buf, 1);
+    free(buf);
+
+    buf = rpmGenPath("%{u2p:%{_builddir}}", "%{buildsubdir}", "SPECPARTS");
```

Comment:
What's with this u2p thing? %_builddir is just a plain old directory, we really don't want any http:// or ftp:// handling in there.

Reply:
That's a good question. I could swear there's a line doing a similar thing just above for the buildroot. Turns out there isn't and `u2p` is only used in `macros.in`. Anyway, it gone now.

Reply:
Oh, I hadn't realized we have u2p being used someplace still. Lets get rid of those (but not in scope for this of course)